docs/dev/connectors/kafka.md

```diff
@@ -73,21 +73,33 @@ For most users, the `FlinkKafkaConsumer08` (part of `flink-connector-kafka`) is
         <td>This connector supports <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-32+-+Add+timestamps+to+Kafka+message">Kafka messages with timestamps</a> both for producing and consuming.</td>
     </tr>
     <tr>
-        <td>flink-connector-kafka-0.11_2.11</td>
+        <td>flink-connector-kafka-0.11{{ site.scala_version_suffix }}</td>
         <td>1.4.0</td>
         <td>FlinkKafkaConsumer011<br>
         FlinkKafkaProducer011</td>
         <td>0.11.x</td>
         <td>Since 0.11.x Kafka does not support scala 2.10. This connector supports <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-98+-+Exactly+Once+Delivery+and+Transactional+Messaging">Kafka transactional messaging</a> to provide exactly once semantic for the producer.</td>
     </tr>
     <tr>
-        <td>flink-connector-kafka_2.11</td>
+        <td>flink-connector-kafka{{ site.scala_version_suffix }}</td>
         <td>1.7.0</td>
         <td>FlinkKafkaConsumer<br>
         FlinkKafkaProducer</td>
         <td>>= 1.0.0</td>
-        <td>This Kafka connector attempts to track the latest version of the Kafka client. The version of the client it uses may change between Flink releases. Modern Kafka clients are backwards compatible with broker versions 0.10.0 or later. However for Kafka 0.11.x and 0.10.x versions, we recommend using dedicated flink-connector-kafka-0.11 and link-connector-kafka-0.10 respectively.</td>
-        </tr>
+        <td>
+        This universal Kafka connector attempts to track the latest version of the Kafka client.
+        The version of the client it uses may change between Flink releases.
+        Modern Kafka clients are backwards compatible with broker versions 0.10.0 or later.
+        However for Kafka 0.11.x and 0.10.x versions, we recommend using dedicated
+        flink-connector-kafka-0.11 and link-connector-kafka-0.10 respectively.
+        <div class="alert alert-warning">
+          <strong>Attention:</strong> as of Flink 1.7 the universal Kafka connector is considered to be
+          in a <strong>BETA</strong> status and might not be as stable as the 0.11 connector.
+          In case of problems with the universal connector, you can try to use flink-connector-kafka-0.11{{ site.scala_version_suffix }}
+          which should be compatible with all of the Kafka versions starting from 0.11.
```

Comment:
Here, maybe not. Do you remember the producer transaction uses reflect to access the inner private field ([sequenceNumbers 0.11 -> nextSequence 1.0+](https://github.com/apache/kafka/commit/5d2422258cb975a137a42a4e08f03573c49a387e#diff-f4ef1afd8792cd2a2e9069cd7ddea630)) ?

Reply:
This is a good point, but as far as I know this is related only if someone would want to upgrade `kafka.version` of the `flink-connector-kafka`. There were couple of users that were successfully using `flink-connector-kafka-0.11` against Kafka 1.0 and it should work for the same reasons why `universal` connector should work with any Kafka version `0.11+`